### PR TITLE
Fixed Discord party invites being blocked

### LINKED ISSUE #20598: Discord Game Invite Doesn't work (Regression, Playtest)
2023-01-03 16_27_04-Window
^Is missing on playtest

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/20598

### CODE REVIEW

File: OpenRA.Mods.Common/DiscordService.cs [hunk 1]
```diff
@@ -189,18 +195,8 @@ void SetStatus(DiscordState state, string details = null, string secret = null,
...
 				client.UpdatePartySize(players, slots);
-				return;
-			}
-
-			client.UpdateParty(new Party
-			{
-				ID = Secrets.CreateFriendlySecret(new Random()),
-				Size = players,
-				Max = slots
-			});
```
Comment (PunkPun) [inline OpenRA.Mods.Common/DiscordService.cs]: Why do you call this dead code?

Comment (PunkPun) [reply OpenRA.Mods.Common/DiscordService.cs]: I guess it no longer crashes? 1a77f7320bf3ddd2c0a9cb44ed10aad77dbeb840

Comment (Mailaender) [reply OpenRA.Mods.Common/DiscordService.cs]: In my tests this part of the code is never hit, but I better keep it then.

**Review by PunkPun — APPROVED** [review]
Fixes the issue for me

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/4c39a7515563e1ce68f55d9a39527b604b8be865...21da644fbbcad311e8fcd0e5576e32c7bcf4d9cd)

https://github.com/OpenRA/OpenRA/commit/43d7f9c6dfb741f2b297a093143fc00e77f235cf
https://github.com/OpenRA/OpenRA/commit/a397670dce2d226af93ec2f00ea77039467fba79

**Pull request merged**